codalab_service.py

```diff
@@ -256,8 +262,8 @@ def has_callable_default(self):
     # Worker manager
     CodalabArg(
         name='worker_manager_type',
-        help='Type of worker manager (e.g., aws-batch, azure-batch, slurm); only aws-batch supported right now',
-        default='aws-batch',
+        help='Type of worker manager (azure-batch or aws-batch)',
+        default='azure-batch',
```

Comment:
```suggestion
        default='aws-batch',
```

Better to keep it `aws-batch` by default for backwards compatibility?

Reply:
I don't think we need to maintain backwards compatibility. Before the code changes in this PR, starting AWS Worker Managers via `codalab_service` was broken.